figures/sites.py

```diff
@@ -241,3 +241,25 @@ def student_modules_for_course_enrollment(ce):
     Relies on the fact that course_ids are globally unique
     """
     return StudentModule.objects.filter(student=ce.user, course_id=ce.course_id)
+
+
+def site_certificates(site):
```

Comment:
**Nit:** This function doesn't seem to be both unused and unrelated to the pull request. Was it added by mistake?

Reply:
Not added by mistake, just a little extra that I remembered I wanted in there as I was refactoring